src/bootstrap/compile.rs

```diff
@@ -273,7 +273,10 @@ fn copy_sanitizers(
     compiler: &Compiler,
     target: Interned<String>,
 ) -> Vec<PathBuf> {
-    let runtimes: Vec<native::SanitizerRuntime> = builder.ensure(native::Sanitizers { target });
+    let runtimes: Vec<native::SanitizerRuntime> = builder.ensure(native::Sanitizers {
+        target,
+        channel: INTERNER.intern_string(builder.config.channel.clone()),
```

Comment:
This can be retrieved inside the sanitizer, we only ever have one channel at a time so there's no need to stick it in the struct.

Reply:
Not sure why I did that in the first place, fixed this